app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -297,7 +295,7 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
         rootView.timeoutInfoTextView.text =
             getString(R.string.ticket_timeout_info_message, event.orderExpiryTime.toString())
 
-        val timeLeft: Long = if (attendeeViewModel.timeout == -1L) COUNT_DOWN_TIME * 60 * 1000L
+        val timeLeft: Long = if (attendeeViewModel.timeout == -1L) event.orderExpiryTime * 60 * 1000L
```

Comment:
This was moved to settings, so it won't be available in event

Reply:
But how it is working currently. I've tested it, frontend is also using the same endpoint.

Reply:
Working on dev or prod?

Reply:
https://github.com/fossasia/open-event-server/pull/6083

Reply:
> Working on dev or prod?

production, while creating an event, time is asking form the organizer.

Reply:
Yeah, so again, this was moved to settings, so it won't be available in event. (10 days ago)

Always consider dev to be latest. It'll break when server is updated.

That's why it's asked for client devs to only use dev instance. You must be ready for any breaking change being applied on server. If you'll use prod environment, how will you deal with new changes?

Reply:
> Always consider dev to be latest. It'll break when server is updated.

Okay, I should also check with the dev instance. It was during creating the meetup event. Will update.

Reply:
Only admin can reterive order expiry time - 
![image](https://user-images.githubusercontent.com/27783786/61105018-2851a600-a496-11e9-8d06-71709aeafb4b.png)

Reply:
Open an issue to allow clients to see order expiry time. If this is true, then I guess timer on frontend is wrong as well

Reply:
Will fetch setting and test +1
Should I fetch setting at the starting of the app and store into the database or fetch every time when required?

Reply:
Save into DB and when retrieving, first load from DB and then send a request to refresh

Reply:
> Save into DB and when retrieving, first load from DB and then send a request to refresh

One problem will there if I load time from the database first and after that refresh it from the server then count down timer will start again by observing new value.

Reply:
Currently, I'm directly fetching settings at the time of ordering tickets. Updated.

Reply:
You will not show it again, it'll be shown next time user opens the order screen. If even you are directly fetching, save in DB